setup.py

```diff
@@ -20,7 +20,14 @@
 
 def _get_cpu_count():
     if platform.system() != "Windows":
```

Comment:
while you're at it, since the function is becoming bigger, I'd suggest getting the windows-only lines closer, like so
```python
if platform.system() == "Windows":
    return 0
... # the rest of the function goes here
```